### PR TITLE
Add note about resources if using different namespace

### PR DESCRIPTION
**Pull request by piyush-garg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -385,6 +385,13 @@ Create the above pipeline resources via the OpenShift web console or by running
 $ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/pipeline/resources.yaml
 ```
 
+> **Note** :-
+>
+>If you are not in `pipelines-tutorial` namespace, and using other namespace for tutorial, please make sure you update
```
Comment (chmouel) [inline README.md]:
```suggestion
>If you are not into the `pipelines-tutorial` namespace, and using another namespace for the tutorial steps, please make sure you update the
```

File: README.md [hunk 2]
```diff
@@ -385,6 +385,13 @@ Create the above pipeline resources via the OpenShift web console or by running
...
+frontend and backend image resource to correct url with your namespace name like
```
Comment (chmouel) [inline README.md]:
```suggestion
frontend and backend image resource to the correct url with your namespace name like so : 
```

**Review by chmouel — CHANGES_REQUESTED** [review]

**Comment by piyush-garg** [comment]
@chmouel Chnages done

**Comment by chmouel** [comment]
/lgtm
/approve

Thanks 👍🏼🔥

**Pull request merged**